### PR TITLE
Updated to support new Test API

### PR DESCRIPTION
**Pull request by mattcorr**
Have updated the sample code to reflect the new API that I stood up as the test backend.

There were some slight differences in how the API worked, so these changes have been reflected.

### CODE REVIEW

File: Edition-01/Starwars-Demo/src/StarWarsData.ps1 [hunk 1]
```diff
@@ -106,23 +108,23 @@ function Get-SWPerson {
 
     if ($null -ne $person.Error)
```
Comment (TheFreeman193) [inline Edition-01/Starwars-Demo/src/StarWarsData.ps1]:
```suggestion
    if ($null -eq $person)
```
Looks like a valid request for a nonexistent person returns an empty array with your new API, which `Invoke-StarWarsApi` returns as null.

File: Edition-01/Starwars-Demo/Starwars.md [hunk 2]
```diff
@@ -5,7 +5,8 @@
 This repo contains a sample **PowerShell** script and various **Pester** tests.
 
 The purpose of the repo is to demonstrate Pester usage for both standard and mocked tests.
-The script wraps a Star Wars API site located here: https://swapi-deno.azurewebsites.net
+The script wraps a Star Wars API site located here: https://mc-starwars-data.azurewebsites.net
```
Comment (TheFreeman193) [inline Edition-01/Starwars-Demo/Starwars.md]:
```suggestion
The script wraps a Star Wars API site located here: <https://mc-starwars-data.azurewebsites.net>
```
Wrapped a bare URL.

**Review by TheFreeman193 — CHANGES_REQUESTED** [review]
Looks great on the whole! The only issue I can find in the code is with `Get-SWPerson` which is still using the result handling for the old API.

The Starwars.md file has a couple of markdown formatting glitches:

+ Bare URL on L8
+ No spaces around codeblock on L32-34. You can indent the entire codeblock to have it be part of the list.

Finally, might it be better to rename Starwars.md to README.md so it is displayed automatically?

File: Edition-01/Starwars-Demo/src/StarWarsData.ps1 (hunk 1 above)
Comment (mattcorr) [reply Edition-01/Starwars-Demo/src/StarWarsData.ps1]: Thanks for spotting that! This has been fixed.

File: Edition-01/Starwars-Demo/Starwars.md (hunk 2 above)
Comment (mattcorr) [reply Edition-01/Starwars-Demo/Starwars.md]: Thanks! I have made the change

**Comment by mattcorr** [comment]
Thanks for the good feedback @TheFreeman193  :)

**Comment by TheFreeman193** [comment]
@mattcorr No worries :grin: LGTM!

**Review by TheFreeman193 — APPROVED** [review]

**Pull request merged**